src/sql/workbench/browser/modal/dialogHelper.ts

```diff
@@ -23,7 +23,6 @@ export function appendRow(container: HTMLElement, label: string, labelClass: str
 	labelContainer.style.display = 'flex';
 
 	if (title) {
-		labelContainer.classList.add("codicon");
```

Comment:
I thought about that, but it would become a very generic class, and it may contradict any other component classes unless it went into standard classes of codicon or something.

Reply:
Curious what you mean by it "changes the entire design". What design is that? And what hacks are you talking about for the tooltip?

Reply:
I meant the Custom Options tooltip current design would need rework, making sure descriptions can align well in connection dialog limited width. Currently we're using 'title' attribute, which provides in-built tooltip that is much lighter on UI, but with the other design tooltip is a component that is controlled on hover, it's not HTML's tooltip/title. 

If you want to spend time on that and think it's worth the effort, feel free to go for it, but as far it meets requirements, we shouldn't need to spend time on redoing it.